examples/slu/speech_intent_slot/requirements.txt

```diff
@@ -0,0 +1,6 @@
+progress==1.5
+tabulate==0.8.7
+textdistance==4.1.5
+jiwer==2.0.0
+jsonlines
```

Comment:
This is a BSD licensed library - please replace with some alternative. We cannot import or install such libraries inside nemo.